pkg/autoscaler/autoscaler/pdplan_autoscaler.go

```diff
@@ -85,8 +86,35 @@ func (am *autoScalerManager) deleteAutoscalingClusters(tc *v1alpha1.TidbCluster,
 }
 
 func (am *autoScalerManager) updateAutoscalingClusters(tac *v1alpha1.TidbClusterAutoScaler, groups []string, groupTcMap map[string]*v1alpha1.TidbCluster, groupPlanMap map[string]pdapi.Plan) error {
-	// TODO in next PR
-	return nil
+	var errs []error
+	for _, group := range groups {
+		actual, oldTc, plan := groupTcMap[group].DeepCopy(), groupTcMap[group], groupPlanMap[group]
+		component := plan.Component
+
+		if !checkAutoscalingComponent(tac, component) {
+			continue
+		}
+
+		switch component {
+		case v1alpha1.TiKVMemberType.String():
+			if actual.Spec.TiKV.Replicas == int32(plan.Count) {
```

Comment:
Maybe we don't need it?

Reply:
we can reduce the number of API calls to k8s if they are the same

Reply:
make sense